bagel/cli.py

```diff
@@ -245,13 +247,45 @@ def bids(
             # so the API can still find the session-level information.
             # This should be revisited in the future as for these cases the resulting dataset object is not
             # an exact representation of what's on disk.
-            session_label = "nb01" if session is None else session
+            if session is None:
+                session_label = "nb01"
+                # Get absolute path, following symlinks, as posix string
+                session_path = (
+                    Path(
+                        # TODO: Once bug in fetching subject directories with no session layers is resolved,
+                        # switch to using layout.get() snippet below to fetch subject path.
+                        os.path.join(bids_dir, f"sub-{bids_sub_id}")
+                        # layout.get(
+                        #     subject=bids_sub_id,
+                        #     target="subject",
+                        #     return_type="dir",
+                        # )[0]
+                    )
+                    .resolve()
+                    .as_posix()
```

Comment:
`.as_posix` does provide a safeguard for a Windows system to keep file separators consistent in the pheno_bids.jsonld, but it also just serves the purpose of converting the Path to a string with forward slashes (which works regardless of the system type). Since the session path needs to be a string anyways in our data model, this just performs a two-in-one 🙂

Reply:
ah, I see. Thanks for explaining!